### DISCUSSION TITLE
Use the upper left side of the song select screen to use it as a new way to show more beatmap info

### DISCUSSION THREAD

**Discussion by caulmseh** [ppy/osu#14868] [Ideas]
Allow beat mappers to utilize the upper left side to show more graphics like they do in the description of a beat map instead of seeing the beat map photo in three parts of the UI (bg blur, song list, upper left side screen)

![image](https://user-images.githubusercontent.com/51788724/135000417-eb969f4b-37b4-4c58-a951-8391390e11ae.png)
![image](https://user-images.githubusercontent.com/51788724/135000791-a69a6fa8-6357-4eb4-b977-a0a05416c24d.png)

Also, I just noticed, the description only shows the difficulty name, not the description that is shown on the website. Is this a bug or a placeholder for future implementation?

**Comment by peppy** [comment]
Placeholder. Needs bbcode parsing.
